src/database.rs

```diff
@@ -1,41 +1,70 @@
-extern crate r2d2;
-extern crate r2d2_redis;
-
-use std::ops::Deref;
-use std::error::Error;
-use self::r2d2_redis::RedisConnectionManager;
-use redis;
-use iron::prelude::*;
-use iron::typemap::Key;
-use persistent::Read;
-use juniper;
-
-pub type Pool = r2d2::Pool<RedisConnectionManager>;
-
-pub fn init_pool() -> Result<Pool, Box<Error>> {
-    let manager = RedisConnectionManager::new("redis://127.0.0.1/")?;
-    Ok(r2d2::Pool::new(manager)?)
+table! {
+    album (id) {
+        id -> Text,
```

Comment:
If you change the IDs in the SQL file, change them here as well (and in the model structs)

Reply:
This file is generated. There should be a build step to validate this file is up to date. #14